src/pyrokinetics/local_geometry/local_geometry.py

```diff
@@ -528,25 +527,13 @@ def get_bunit_over_b0(self, ntheta: int = 256):
 
         """
 
-        theta = np.linspace(0, 2 * pi, ntheta)
+        theta = np.linspace(0, 2 * pi, ntheta, endpoint=False)
+        dtheta = np.diff(theta, append=theta[0] + 2 * np.pi)
 
         R, Z = self.get_flux_surface(theta=theta)
+        dRdtheta, dRdr, dZdtheta, dZdr = self.get_RZ_derivatives(theta)
 
```

Comment:
In f_psi we get dRdtheta and dZdtheta as properties of self, is there a reason we don't do that here? Calculating dL appears in a few routines -- can we pull it out into a helper?

Reply:
This was to ensure that we always use the same `ntheta` when calculating the `Bunit/B0` as they `ntheta` could vary depending on the resolution used by the code. With the previous method this caused more variability but now it is less of a concern.

It was important to get this consistent regardless of resolution as `Bunit/B0` was used to convert between simulations so should ideally be resolution independent

Reply:
Pulled out `dLdtheta`